diagnostic.md

```diff
@@ -55,6 +72,27 @@ How would we reference each of the following categories of elements?
 
 <!-- your answer starts here -->
 
+I'm a little confused by the wording of this question. It seems to want the way
+css refers to classes/id's already created in an html file, so I'm going to
+answer in css.
+
+```CSS
+.big {
+
+}
+
+#contentPane {
+
+}
+
+.important .red {
```

Comment:
`.important.red` would apply the styling to elements that have BOTH of these classes. With the space, you're saying apply the styles to both the `.important` and `.red` class. Big difference.